can/bus.py

```diff
@@ -164,7 +173,13 @@ def send(self, msg, timeout=None):
         """
         raise NotImplementedError("Trying to write to a readonly bus?")
 
-    def send_periodic(self, msgs, period, duration=None, store_task=True):
+    def send_periodic(
+        self,
+        msgs: Union[Sequence[can.Message], can.Message],
```

Comment:
Would you mind removing the duplicate type signature from the doc string as you add them as annotations? 

We should be able to ensure the docs still provide all the info with https://github.com/agronholm/sphinx-autodoc-typehints

Reply:
Yeah, I had that working, but I didn't want to add too much to this PR. I'll create a separate one that adds the new dependency.

Reply:
Great work, man!